tests/integ/scala/test_function_suite.py

```diff
@@ -2229,6 +2229,7 @@ def test_as_object(session):
 
 
 def test_timestamp_tz_from_parts(session):
+    session.sql("alter account set timezone = 'America/Los_Angeles'").collect()
```

Comment:
This probably won't work in stored proc, or should not be used for remote test. What is the failure that we are trying to solve?

Reply:
is there anything else we could do here?
a sample failed test: https://ci-dev-133.int.snowflakecomputing.com/job/RT-PC-TestWorker/29487/testReport/junit/(root)/t_python_sp/snowpark_scala_test_function_suite_sql/

Reply:
You could add an `alter session` in the regression test itself :)

Reply:
oh yeah, that makes more sense